### ISSUE TITLE
wz-next function is called before canexit validation

### ISSUE THREAD

**Issue by lindamarieb** [mgonto/angular-wizard#196]
I have a wz-next function and a canexit function for my wizard step. The wz-next function is called before the canexit function. Surely this is the incorrect order?

**Comment by jacobscarter** [comment]
Yes that does not sound right, can you post some sample code? In our example page things work and fire as expected.
